src/main/resources/grammar.hgr

```diff
@@ -269,7 +269,7 @@ grammar {
     $cmd_param = :cmd_param_start list($cmd_param_kv) $e :cmd_param_end -> CommandParameter(attributes=$1, expr=$2)
     $cmd_param_kv = :cmd_attr_hint :identifier :equal $e -> CommandParameterAttr(key=$1, value=$3)
     $outputs = :output :lbrace list($output_kv) :rbrace -> Outputs(attributes=$2)
-    $output_kv = $type_e :identifier :equal $e -> Output(type=$0, var=$1, expression=$3)
+    $output_kv = $type_e :identifier :equal $e -> Output(type=$0, name=$1, expression=$3)
```

Comment:
question: how is "Output" resolved here?  (e.g. where did the renaming of var -> name occur that necessitated this?)